### ISSUE TITLE
p-slider doesn't pass Accessibility Insights for Web Fastpass in some cases

### ISSUE THREAD

**Issue by StevenLHale** [primefaces/primeng#11725]
### Describe the bug

We are using p-slider and are failing Fastpass aria-allowed-attr rule.  Looks like the p-slider-handle span doesn't allow aria-valuemin, etc. unless it has a role set to support those attributes, such as "slider" role, which also requires aria-label of some sort.  As workaround we set the role and aria-label in code which allows Fastpass to pass.  I looked for a way to set it in the html but didn't find it.

<html>
<body>

Path | .zoom-slider-wrapper[tooltipposition="left"][_ngcontent-app-id-c558=""] > p-slider > .zoom-slider.p-slider.p-slider-vertical > .p-slider-handle[aria-valuemin="-0.5"][aria-valuemax="0.7"]
-- | --
<span class="p-slider-handle ng-star-inserted" tabindex="0" aria-valuemin="-0.5" aria-valuenow="0" aria-valuemax="0.7" style="bottom: 41.6667%;"></span>
Fix the following:ARIA attributes are not allowed: aria-valuemin="-0.5", aria-valuenow="0", aria-valuemax="0.7"

</body>
</html>

### Environment

Windows, Angular

### Angular version

13.3.2

### PrimeNG version

13.2.1

### Build / Runtime

TypeScript

### Language

TypeScript

### Node version (for AoT issues node --version)

14.19.0

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
